windows_docker_resources/Dockerfile.foxy

```diff
@@ -78,11 +78,11 @@ RUN 7z.exe x C:\TEMP\opencv.zip -aoa -oC:\
 RUN 7z.exe x C:\TEMP\OpenSplice.zip -aoa -oC:\opensplice
 
 # Environment setup
-ENV OPENSSL_CONF C:\OpenSSL-Win64\bin\openssl.cfg
+ENV OPENSSL_CONF "C:\Program Files\OpenSSL-Win64\bin\openssl.cfg"
```

Comment:
Strange, on my native Windows machine, the OpenSSL installer defaults to ` C:\OpenSSL-Win64` as the install root.

Reply:
Do you have an earlier version of OpenSSL installed? It might be looking at the previous installation path?

Reply:
I re-located the old installation of OpenSSL; but it's possible there was still some user-data floating around.